docs/user-feedback/users.md

```diff
@@ -0,0 +1,9 @@
+# Users
```

Comment:
Can there be instructions for leaving feedback if someone wants to take the initiative to read through outside of the guided walkthroughs?

Reply:
I've updated the README to define a "Self-Guided Feedback" process